From ee07a4c3c8fcbcc7c05c7cec7f97b6e6690cf54a Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sun, 26 Mar 2017 20:24:16 +0200
Subject: [PATCH 1/8] Make tidy not convert character entities to their UTF-8
 form

The output documents do not have any encoding specified, so programs
handling them will assume ASCII, and when finding UTF-8 characters will
recode those, producing garbage on output.
---
 copyright-format/Makefile | 2 +-
 debconf_spec/Makefile     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/copyright-format/Makefile b/copyright-format/Makefile
index 6d26018..31d6e7a 100644
--- a/copyright-format/Makefile
+++ b/copyright-format/Makefile
@@ -3,7 +3,7 @@ all: copyright-format-1.0.txt.gz copyright-format-1.0.html
 copyright-format-1.0.html: copyright-format-1.0.xml html.dsl
 	openjade -V nochunks -t sgml -d html.dsl \
 		/usr/share/xml/declaration/xml.dcl $< > $@
-	-tidy -q -i -m -f /dev/null $@
+	-tidy -raw -q -i -m -f /dev/null $@
 
 copyright-format-1.0.txt: copyright-format-1.0.html
 	links -dump $< | perl -pe 's/[\r\0]//g' > $@
diff --git a/debconf_spec/Makefile b/debconf_spec/Makefile
index cdfd135..bc6614d 100644
--- a/debconf_spec/Makefile
+++ b/debconf_spec/Makefile
@@ -3,7 +3,7 @@ all: debconf_specification.txt.gz debconf_specification.html
 %.html: %.xml html.dsl
 	openjade -V nochunks -t sgml -d html.dsl \
 		/usr/share/xml/declaration/xml.dcl $< > $@
-	-tidy -q -i -m -f /dev/null $@
+	-tidy -raw -q -i -m -f /dev/null $@
 
 %.txt: %.html
 	links -dump $< | perl -pe 's/[\r\0]//g' > $@
-- 
2.12.2.564.g063fe858b8

